pliops/kv_pair_simple_serializer.h

```diff
@@ -8,7 +8,7 @@ namespace Replicator {
 class KvPairSimpleSerializer : public IKvPairSerializer {
 public:
     void Serialize(const char* key, uint32_t key_size, const char* value, uint32_t value_size, std::vector<char>& buffer) override;
```

Comment:
Let's add RepStatus  to Serialize as well

Reply:
done